best-practices.md

```diff
@@ -399,7 +402,7 @@ ended up doing. Following these recommendations makes for more legible catalogs,
 if you follow these recommendations.
 
 1. Root documents (Catalogs / Collections) should be at the root of a directory tree containing the static catalog.
-2. Catalogs that are not also Collections should be named `catalog.json` and Collections should be named `collection.json`.
+2. Catalogs should be named `catalog.json` and Collections should be named `collection.json`.
```

Comment:
Why the removal?

Reply:
Because no catalogs are also collections

Reply:
Oh yeah, I automatically still assume that. Will take some time...